test/Concurrency/experimental_feature_strictconcurrency.swift

```diff
@@ -1,3 +1,4 @@
+// expected-error{{new Swift parser generated errors for code that C++ parser accepted}}
```

Comment:
I am working on, but have not yet completed, the Swift parser implementation, so any merge is blocked upon that, at which point I will remove the erroneous expected errors from this test.